android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/adding/AddingFragment.java

```diff
@@ -270,6 +299,7 @@ private void createDatePicker() {
         datePicker.addOnPositiveButtonClickListener(selection -> {
             Format formatter = new SimpleDateFormat("dd.MM.yyyy", Locale.GERMANY);
```

Comment:
I dont get the layout error, but maybe it is also due to the selected smartphone

Reply:
@XSnelliusX  Do you have any other requests changes ?

Reply:
It is due to the selected smartphone. I don't think this has to be tackled in this ticket as it might be complicated to get the correct length of the layout. Let's keep it the way it is so that it can be merged soon